trie/hexary.py

```diff
@@ -197,8 +200,23 @@ def get_node(self, node_hash):
 
         return node
 
-    @staticmethod
-    def _node_to_db_mapping(node):
+    def node_to_db_mapping(self, node):
```

Comment:
I think this should remain an internal interface (with the `_` prefix). This method could be `_node_to_db_mapping` and the other: `_create_node_to_db_mapping`.